news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,101 @@
+
+## High refresh rate LCD monitors (2012)
+
+Around mid 2012 first LCD high refresh rate monitors were released with the rates as high as 144hz. Even before LCD monitors were a thing CRT's had the capability of refreshing up to 100hz in most cases (and even higher for high-end models), however with the introduction of high refresh rate LCD panels this technology slowly became much easier to adopt, and is now a staple of any modern gaming setup.
+
+As osu! is a game with high amount of moving elements no matter the game mode (as shown below), high refresh rate monitors quickly became the next most-sought-after commodity piece of gear. While being an upgrade in comparison to 60hz monitors, and some people swearing by high refresh rate monitors helping them read higher approach rates & scroll speeds, it did not stop players who could not afford them from rising through the ranks and some of them even becoming top players such as [FGSky](https://osu.ppy.sh/users/2094566), regardless of the refresh rates of their monitor.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="soonTM" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Touch screen's rise and fall (2015-2017)
+
+The earliest mentions of using touch screens as an input device came as soon as the game was released, again due to osu!'s roots as an *Osu! Tatakae! Ouendan* clone. Back then touch screens on PCs were not as prevalent, and most touch screen devices were "Tablet PCs", which were laptops that had a keyboard that can be rotated, folded, or outright detached, along with a touchscreen.
+
+It would take many years for the community to see the true potential of using touch screens in osu!, but starting in 2015 we started seeing glimpses, and eventually the peak of the play style in 2017 with the two players noted below.
+
+### Kuvster (2015)
+
+In 2015, [Kuvster](https://osu.ppy.sh/users/4896624) had a unique opportunity — snag the pp record under a very specific set of circumstances. Back in the day "TAG4" maps were [approved](/wiki/Beatmap/Category#approved), hence you were able to get pp on them. They weren't playable for a regular player, but if you had a touch screen and were apt with it, you were eligible to grab the pp record, which is what they did:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/jcHiuNcwsiM" frameborder="0" allowfullscreen></iframe>
+</div>
+
+The 577pp record was short-lived though, as after this all TAG4 maps were exempt from giving pp, and in 2019 TAG4 maps were moved to the [Loved category](/wiki/Beatmap/Category#loved). This however is not the event what "killed" touch screen, its demise would come only 2 years later.
```

Comment:
```suggestion
The 577pp record was short-lived though, as after this all "TAG4" beatmaps were exempt from giving pp, and in 2019 they were moved to the [Loved category](/wiki/Beatmap/Category#loved). This however is not the event that "killed" the touch screens, its demise would come only 2 years later.
```

Reply:
ill apply this but `the touch screen` would correct as its referring to the play style in its entirety and not like multiple touch screens